lerobot/common/utils/utils.py

```diff
@@ -228,3 +239,102 @@ def is_valid_numpy_dtype_string(dtype_str: str) -> bool:
     except TypeError:
         # If a TypeError is raised, the string is not a valid dtype
         return False
+
+
+class TimerManager:
+    """
+    Lightweight utility to measure elapsed time.
+
+    Examples
+    --------
+    >>> timer = TimerManager("Policy", log=False)
+    >>> for _ in range(3):
+    ...     with timer:
+    ...         time.sleep(0.01)
+    >>> print(timer.last, timer.fps_avg, timer.percentile(90))
+    """
```

Comment:
could be nice to have the result

To be consistent with code base use plain python back-quote
```python

Reply:
COuld we have an example where we dont use `with` ?

Reply:
Done